package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "postinstall": "npm run build",
     "build": "webpack --config webpack.config.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch"
```

Comment:
I don't think we should, I just do this because I forgot the command :smile: , but if you want to remove, that's ok. Do you want?